webapp/channels/src/packages/mattermost-redux/src/action_types/channels.ts

```diff
@@ -91,4 +91,7 @@ export default keyMirror({
     INCREMENT_FILE_COUNT: null,
 
     GM_CONVERTED_TO_CHANNEL: null,
+
+    RECEIVED_IS_DM_RESTRICTED: null,
```

Comment:
similarly, renaming `RECEIVED_IS_DM_RESTRICTED` to `RECEIVED_IS_DM_GM_RESTRICTED` would be consistent.